src/components/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,13 @@
+import { Tooltip as AntdTooltip, TooltipProps } from 'antd'
+
+export type Props = TooltipProps
```

Comment:
Could we use `Expand` here kinda like in the other ones that are implemented? It helps a lot with hover introspection in code editors, and should be functionally the same